json/items.json

```diff
@@ -5,8 +5,8 @@
     "dname": "Blink Dagger",
     "qual": "component",
     "cost": 2250,
-    "desc": "Active: Blink - Teleport to a target point up to 1200 units away.  If damage is taken, Blink Dagger cannot be used for 3.0 seconds.",
-    "notes": "Self-casting will cause you to teleport in the direction of your team's fountain.<br />If you used Blink to teleport to a distance over the maximum range, you'll be teleported 4/5 of the maximum range instead.",
+    "desc": "Active: Blink - Teleport to a target point up to 1200 units away.  If damage is taken, Blink Dagger cannot be used for 3 seconds.",
```

Comment:
If this file is generated via script, remove it from the `json` directory (only keep the copy in `build`)